rust/worker/src/execution/operator.rs

```diff
@@ -37,6 +38,7 @@ pub(crate) type TaskMessage = Box<dyn TaskWrapper>;
 #[async_trait]
 pub(crate) trait TaskWrapper: Send + Debug {
     async fn run(&self);
+    fn getTracingContext(&self) -> Option<tracing::Id>;
```

Comment:
I think this is a great design, if we could port it to the Wrapper in system/ instead of here I think that would be better? That way all messages get tracing. The task ultimately gets wrapped by that.